front-end/e2e/jest.config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   preset: "jest-puppeteer",
-  testTimeout: 8000,
+  testTimeout: 12000,
```

Comment:
This will result in a dramatic slowdown of failing tests. It is really not acceptable to impact 100% of the students for the unknown number of students with a resource constrained machine. 

A better solution is to explain the setting and to override when running their tests locally e.g. --testTimeout or change the file themselves if they really need it, or rely on GH to run the tests for them if they are really that resource constrained.  

I can make this change or you can, please let me know how you would like to proceed.

Reply:
I agree, what you propose sounds like a better solution. Please change as you see fit.